dbtmetabase/models/config.py

```diff
@@ -0,0 +1,30 @@
+from dataclasses import dataclass, field
+from typing import Optional, Iterable, Union
+
+
+@dataclass
+class MetabaseConfig:
```

Comment:
Likely last thing, but what's the purpose of essentially double prefixing by having `metabase_` for every field in an object that's already prefixed `Metabase`? Same with `dbt_` and `Dbt` below. Can we drop these field prefixes?

Reply:
agreed, done